uclaradio/uclaradio.com
# Cleaned blog-project and included new blog posts

## Types of changes


## Purpose

Need to cleanup old files and repetitive code before merging to master. Mia improved the ML categorizing script to better categorize "Sports", "Comedy", "Entertainment", and "UCLA Radio". I included that in this PR too.

## Approach

Mia updated the categorizing script and manually tagged uncategorized posts. Her original input came from the JSON object generated by `app/database/scripts/addTumblrPosts.js`, which fetches all the Tumblr posts except for the 2 most recent posts. They weren't added in time, but I'm going to manually add the 2 most recent posts (both about "The Menu") to Keystone. Her updated categorizing script is included as `app/database/scripts/augment_training_set.py`. 

I also refactored `parseTag` to use her refined categories as `parseCategory`. 

## Testing

1. Reinstall the package.json because I updated the `fill-blog-db` script: `yarn`
2. Run mongod: in one terminal: `mongod`
3. Delete the blogposts object from the mongo shell: in a second terminal: `mongo`
`use uclaradio`
`db.blogposts.drop()`
`Ctrl+C`
3. Download the [NEW categorized-blogposts.json from Google Drive](https://drive.google.com/file/d/1kT8-k68T_zrUE4TTO-R8owO6IuUt4QVb/view?usp=sharing)
4. Import the new file into the database: `yarn fill-blog-db`
5. Start the server `yarn dev`. Navigate to localhost:3000/blog.
6. Click the "Categories" bar. Click "Sports", "Comedy", "Entertainment", or "UCLA Radio". These should now have lots of results.

## Screenshot(s)

### Filtering with Entertainment and Sports generates a lot more posts
<img width="663" alt="screen shot 2019-03-06 at 11 46 13 am" src="https://user-images.githubusercontent.com/22732325/53909029-814f9000-4005-11e9-9bbe-7143406c80b6.png">

## Checklist
- [x] My branch follows the branch naming scheme of UCLA Radio, and can merge into `master` without error.
- [x] My code follows the code style of this project, and I have linted it to confirm this.

## Link to Issue

[#326](https://github.com/uclaradio/uclaradio.com/issues/326)
